frontend/__tests__/src/fixtures.ts

```diff
@@ -69,7 +69,8 @@ export const BOARD_FIELDS = ['Board', 'Board Id', 'Email', 'Project Key', 'Site'
 export const PIPELINE_TOOL_FIELDS = ['Pipeline Tool', 'Token']
 export const SOURCE_CONTROL_FIELDS = ['Source Control', 'Token']
 
-export const MOCK_BOARD_URL = '/api/v1/boards/jira'
+export const MOCK_BOARD_URL_FOR_VERIFY_JIRA = '/api/v1/boards/Jira'
```

Comment:
Why use Jira, should be jira，
As classic-jira

Reply:
Because we want to make mock url has the same value with real url for board. In boardClient, get method use '/board/Jira' and '/board/Classic_Jira'